OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -223,11 +232,11 @@ else if (permissionList.contains("android.permission.ACCESS_COARSE_LOCATION")) {
                   sendAndClearPromptHandlers(promptLocation, true);
                   startGetLocation();
                } else {
-                  sendAndClearPromptHandlers(promptLocation, false);
+                  sendAndClearPromptHandlers(promptLocation, false, alertDialogTitle, alertDialogMessage);
```

Comment:
Careful, we don't want to display a message if the only issue was the user denied location permission.

Reply:
good catch!